src/qsub.jl

```diff
@@ -68,7 +70,7 @@ function launch(manager::Union{PBSManager, SGEManager}, params::Dict, instances_
 end
 
 function manage(manager::Union{PBSManager, SGEManager}, id::Integer, config::WorkerConfig, op::Symbol)
-    if op == :finalize
+    if op == :interrupt
```

Comment:
Well, it wasn't killing the workers, closing just dumped a bunch of random stuff to no end on STDOUT that I had to Ctrl+Z and kill, Ctrl+C did nothing. In the current implementation op SSHManagers there is no finalize op, just an interrupt so I changed it to this and it kills the worker properly.